src/osemgrep/configuring/Engine_type.ml

```diff
@@ -0,0 +1,5 @@
+module Out = Semgrep_output_v1_t
+
+(* TODO it would be better for this also to be in the atd *)
```

Comment:
would be better also to have
type engine_kind =
 | Oss
 | Pro of pro_flavor
instead of this ugly pair.

Reply:
Good catch; I did this for a reason that no longer applies